bundles/org.openhab.binding.velux/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,181 @@
+
+	<channel-type id="status">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.status.label</label>
+		<description>@text/channel-type.velux.status.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="reload">
+		<item-type>Switch</item-type>
+		<label>@text/channel-type.velux.reload.label</label>
+		<description>@text/channel-type.velux.reload.description</description>
+		<category>Bridge</category>
+	</channel-type>
+
+	<channel-type id="timestamp" advanced="true">
+		<item-type>Number</item-type>
+		<label>@text/channel-type.velux.timestamp.label</label>
+		<description>@text/channel-type.velux.timestamp.description</description>
+		<category>Bridge</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="doDetection" advanced="true">
+		<item-type>Switch</item-type>
+		<label>@text/channel-type.velux.doDetection.label</label>
+		<description>@text/channel-type.velux.doDetection.description</description>
+		<category>Bridge</category>
+	</channel-type>
+
+	<channel-type id="firmware" advanced="true">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.firmware.label</label>
+		<description>@text/channel-type.velux.firmware.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="ipAddress" advanced="true">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.ipAddress.label</label>
+		<description>@text/channel-type.velux.ipAddress.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="subnetMask" advanced="true">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.subnetMask.label</label>
+		<description>@text/channel-type.velux.subnetMask.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="defaultGW" advanced="true">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.defaultGW.label</label>
+		<description>@text/channel-type.velux.defaultGW.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="DHCP" advanced="true">
+		<item-type>Switch</item-type>
+		<label>@text/channel-type.velux.DHCP.label</label>
+		<description>@text/channel-type.velux.DHCP.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="WLANSSID" advanced="true">
+		<item-type>String</item-type>
+		<label>@text/channel-type.velux.WLANSSID.label</label>
+		<description>@text/channel-type.velux.WLANSSID.description</description>
+		<category>Bridge</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="WLANPassword" advanced="true">
+		<item-type>String</item-type>
```

Comment:
This does not sound like a channel to me, so do a lot of the other fields, I will also ask the other maintainers to take a look at the modeling here.

CC: @openhab/2-x-add-ons-maintainers

Reply:
Still open: For now, there are properties in parallel to channels (which are marked as enhanced) for experienced users.

Reply:
@openhab/add-ons-maintainers / @Hilbrand / @kaikreuzer should this be fixed before merging or is it okay as is?